tests/acceptance/features/apiActivities/activities.feature

```diff
@@ -810,6 +810,808 @@ Feature: check activities
       }
       """
 
+
+  Scenario: check activities of destination folder after moving a file into it
```

Comment:
can we check for both here? you can merge the scenarios if there's already one for destination file
```suggestion
  Scenario: check activities of destination file and folder after moving a file into it
```

Reply:
Yeah we can do that :100: